.github/workflows/terraform-apply.yml

```
@@ -63,91 +95,81 @@ jobs:
       - name: Apply 출력 업로드
         uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: tfapply-${{ github.event.inputs.environment }}-${{ github.run_number }}
-          path: terraform/environments/${{ github.event.inputs.environment }}/apply_output.txt
+          name: tfapply-${{ matrix.terraform_environment }}-${{ github.run_number }}
+          path: terraform/environments/${{ matrix.terraform_environment }}/apply_output.txt
 
       - name: 배포 요약 생성
         if: always()
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:11:53: Double quote to prevent globbing and word splitting

(shellcheck)